src/stylesheets/components/_navigation.scss

```diff
@@ -51,6 +51,7 @@
   padding: 2rem;
   width: $sliding-panel-width;
   z-index: $z-index-nav;
+  visibility: hidden;
```

Comment:
This ensures that keyboard focus can't go into the nav menu unless it's visible, either because desktop mode is active, or because the mobile nav is active.